chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SvapController.java

```diff
@@ -0,0 +1,73 @@
+package gov.healthit.chpl.web.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.healthit.chpl.domain.CertificationCriterion;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+import gov.healthit.chpl.exception.ValidationException;
+import gov.healthit.chpl.svap.domain.Svap;
+import gov.healthit.chpl.svap.manager.SvapManager;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Api(value = "svaps")
+@RestController
+@RequestMapping("/svaps")
```

Comment:
Does it need @Loggable which was added to (all?) other controllers in another PR?